templates/default/sssd.conf.erb

```diff
@@ -25,19 +23,8 @@ realmd_tags = manages-system joined-with-samba
 cache_credentials = True
 krb5_store_password_if_offline = True
 default_shell = /bin/bash
+ldap_sasl_authid = <%= @sasl_authid %>$
```

Comment:
@jeremyhahn I removed this. It was in there for compatibility with realm, but it wasn't being used at all. Since we aren't using realm right now I removed it.